src/runner/PlutoRunner.jl

```diff
@@ -259,12 +259,12 @@ function try_macroexpand(mod, cell_uuid, expr)
     end
 
     elapsed_ns = time_ns()
-    expanded_expr = macroexpand(mod, expr_not_toplevel)
+    expanded_expr = macroexpand(mod, expr_not_toplevel)::Expr
```

Comment:
same question about type assertions

Reply:
Good point. For some reason, it couldn't be inferred here. I don't know why exactly. I'm happy to hear better solutions.

For the `CantReturnInPluto.replace_returns_with_error` below, I suspect that the compiler doesn't step into the module, but I'm not sure.

Reply:
Yeah macroexpand can return anything if the expr is a macrocall (not happening in our case because the expr is wrapped in a `:toplevel`):

```julia
julia> macro x()
           nothing
       end
@x (macro with 1 method)

julia> @macroexpand(@x()) === nothing
true
```

Reply:
Yeah macroexpand can return anything if the expr is a macrocall (not happening in our case because the expr is wrapped in a `:toplevel`):

```julia
julia> macro x()
           nothing
       end
@x (macro with 1 method)

julia> @macroexpand(@x()) === nothing
true
```